src/main/resources/reference.conf

```diff
@@ -61,7 +61,7 @@ liquibase {
 # Jupyter proxy server configuration
 proxy {
   jupyterProxyDockerImage = "broadinstitute/openidc-proxy:2.3.1_2"
-  proxyServerName = "jupyter-proxy-server"
+  proxyServerName = "proxy-server"
```

Comment:
Is this change related to docker volume change?

Reply:
no